.gitlab-ci.yml

```diff
@@ -9,6 +9,18 @@ build:
   extends:
     - .build
   stage: build
+  variables:
+    C0_GLOBAL_BASE: /-/telescopium/-/jobs/$CI_JOB_ID/artifacts/dist
+  after_script:
+    - node scripts/local_link_suffixer.js
+    - |
+      echo -e "\e[0Ksection_start:`date +%s`:glpa_summary\r\e[0KHeader of the summary"
+      echo "Documentation preview available at https://code0-tech.gitlab.io${C0_GLOBAL_BASE}/index.html"
```

Comment:
Todo: Add a job to the template for the other projects to also include the documentation preview url